theme/static/js/generic-resource.js

```diff
@@ -150,6 +150,23 @@ function removeExtraMetaTable(table) {
     saveExtraMetadata();
 }
 
+function showRemoveCommentPopup(comment_id_str, has_children_bool) {
+    // this is a hidden HTML element to store the row_id_str
+    $("#delete_comment_id").val(comment_id_str);
+    if(has_children_bool){
+        $("#delete_comment_message").text("This will delete this comment and the threaded reply messages")
+    }
+    else{
+        $("#delete_comment_message").text("This will delete this comment")
+    }
+    $('#deleteCommentDialog').modal('show');
+}
+
+function removeComment() {
+    $("#deleteCommentDialog").modal('hide');
+    window.location = "/comment/delete/" + $("#delete_comment_id").val().trim();
+}
```

Comment:
Is this removeComment() function needed? I see the view function return a http redirect to refresh the whole resource page, so why not trigger the view function url directly in the modal dialog? Are you able to directly use #delete_comment_id value in the delete comment modal dialog?

Reply:
I followed the patterns set by the other modal confirmation boxes.  I think there are probably easier patterns out there, but I think we should stay consistent.